docs/quickstart/visualization/k8s-network-mapper.mdx

```diff
@@ -94,10 +104,11 @@ If you've attached Otterize OSS to Otterize Cloud, you can now also see the [acc
 The access graph reveals several types of information and insights, such as:
 
 1. Seeing the network map for different clusters, seeing the subset of the map for a given namespace, or even &mdash; according to how you've mapped namespaces to environments &mdash; seeing the subset of the map for a specific environment.
-2. Filtering the map to include recently-seen traffic, since some date in the past. That way you can eliminate calls that are no longer relevant, without having to reset the network mapper and start building a new map.
-3. If the intents operator is also connected, the access graph now reveals more specifics about access: understand which services are protected or would be protected, and which client calls are being blocked or would be blocked. We'll see more of that in the next couple of tutorials
+2. Viewing the public internet egress traffic for each pod, including the DNS name and the IPs associated with each outbound request.
+3. Filtering the map to include recently-seen traffic, since some date in the past. That way you can eliminate calls that are no longer relevant, without having to reset the network mapper and start building a new map.
+4. If the intents operator is also connected, the access graph now reveals more specifics about access: understand which services are protected or would be protected, and which client calls are being blocked or would be blocked. We'll see more of that in the next couple of tutorials
```

Comment:
```suggestion
4. If the intents operator is also connected, the access graph now reveals more specifics about access: understand which services are protected or would be protected, and which client calls are being blocked or would be blocked. We'll see more of that in the next couple of tutorials.
```
Pay attention to small deets like this one :)

Reply:
That was preexisting, but I'll fix it.